topi/include/topi/elemwise.h

```diff
@@ -169,5 +171,110 @@ inline Tensor cast(const Tensor& x,
   }, name, tag);
 }
 
+/*!
+* \brief Creates an operation that sum each element of a tensor
+*
+* \param x The input tensor
+* \param num_args The number of arguments
+*
+* \return A Tensor whose op member is the sum operation
+*/
+inline Tensor elemwise_sum(const Array<Tensor>& xs,
+                           int num_args,
```

Comment:
What is the significance of num_args?

Reply:
I add num_args to it to correspond to [python implementation](https://github.com/dmlc/tvm/blob/master/topi/python/topi/tensor.py#L8), but it is not used in there too. If not necessary, I'll remove it both cpp and python. Would you comment on this? 
\cc @tqchen

Reply:
Can be remove safely

Reply:
Ok, thanks!